Makefile

```
@@ -28,21 +28,28 @@ TLS_SETTING := $(if $(filter $(OCI_BIN),podman),--tls-verify=false,)
 GO_BUILD_OPTS ?= CGO_ENABLED=0 GO111MODULE=on
 GO_TAGS ?= -tags no_openssl
 GO_FLAGS ?= -mod vendor
+GOLANGCI_LINT_VERSION ?= v2.7.2
 
 all: lint build
```

checkmake (0.3.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)